account-keys/Cargo.toml

```diff
@@ -45,7 +45,7 @@ mc-util-test-vector = { path = "../util/test-vector" }
 mc-util-test-with-data = { path = "../util/test-with-data" }
 rand = "0.8"
 rand_hc = "0.3"
-tempdir = "0.3"
+tempfile = "3.4"
```

Comment:
Is it possible account-keys doesn't need this at all? If the dependency changed but no rust code changed then that is likely the case...
This might apply to other places.